tools/api-markdown-documenter/src/test/EndToEndTestUtilities.ts

```diff
@@ -0,0 +1,176 @@
+/*!
```

Comment:
The new type hierarchies make it more difficult to share tests directly. As a result, this module has been refactored to share common utilities, and the actual test definitions have been moved into the individual end-to-end test modules.